cns/middlewares/k8sSwiftV2.go

```diff
@@ -55,6 +57,19 @@ func (k *K8sSWIFTv2Middleware) IPConfigsRequestHandlerWrapper(defaultHandler, fa
 		if !req.SecondaryInterfacesExist {
 			return ipConfigsResp, err
 		}
+
+		// ipConfigsResp has infra IP configs -> if defaultDenyACLbool is enabled, add the default deny acl's pn the infra IP configs
+		for i := range ipConfigsResp.PodIPInfo {
+			ipInfo := &ipConfigsResp.PodIPInfo[i]
```

Comment:
Any reason not to do:

```suggestion
		for _, ipInfo := range ipConfigsResp.PodIPInfo {

```

Reply:
Hi, there are 2 reasons why I did not go that route

-  I see the following error: rangeValCopy: each iteration copies 256 bytes (consider pointers or indexing) (gocritic)

 ![image](https://github.com/user-attachments/assets/5f5c6c47-46e5-4c36-a9b2-573239371c11)

- This line of code:  **_, ipInfo := range ipConfigsResp.PodIPInfo** creates a copy of ipInfo and &ipInfo creates a pointer to the copy; however, does not change the slice itself. 

Please feel free to correct me on this, but that was my thinking.